stats/losses/core.py

```diff
@@ -10,6 +10,35 @@ def __init__(self, api_key):
         self.api_key = api_key
         print("created Config")  
 
+class StatsService():
+
+    def __init__(self):
+        API_KEY = os.getenv('RIOT_API_KEY_APP')
```

Comment:
you should consider having a layer to hide config sources (enironment, files, command line).  Will make testing easier and give you more flexibility.

Build the configuration object somewhere else, like with https://pypi.org/project/ConfigArgParse/ or https://github.com/dynaconf/dynaconf, and pass it in to the object here

Reply:
Going to add this to the ToDo, will look into it today